packages/wallets/bitkeep/src/adapter.ts

```diff
@@ -9,10 +10,185 @@ import {
+    readonly supportedTransactionVersions = null;
```

Comment:
Hmm, do you not support `VersionedTransaction` on Solana? This will make your wallet incompatible with most of the major Solana apps.